.changeset/tough-apes-learn.md

```diff
@@ -0,0 +1,7 @@
+---
+"wrangler": minor
+---
+
+R2 event notifications will not support LifecycleDeletion events
```

Comment:
Can you add a type prefix to this line please

```suggestion
fix: R2 event notifications will not support LifecycleDeletion events
```

Reply:
Covered by the changeset in #5294. That change and this are expected to go out in the same release.